code/modules/client/preference_setup/loadout/items/head.dm

```diff
@@ -255,7 +255,7 @@
 /datum/gear/head/hijab
 	display_name = "hijab selection"
 	path = /obj/item/clothing/head/hijab
-	slot = slot_r_ear
+	slot = slot_head
```

Comment:
`/datum/gear/head` already defines `slot` as `slot_head` so you could delete the whole line for the same effect
```suggestion
```

Reply:
Done